setup.py

```diff
@@ -6,7 +6,8 @@
 
 GENERIC_REQ = [
     "six == 1.10.0",
-    "requests == 2.13.0"
+    "requests == 2.13.0",
+    "pyrfc3339 == 1.0"
```

Comment:
nit: adding a trailing comma on the last list item makes diffs like this (very slightly) more readable